### PR TITLE
Add Engadget

### PR DESCRIPTION
**Pull request by soodaayush**

### CODE REVIEW

File: img/e/engadget.com.svg [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+     inkscape:window-y="-9"
+     inkscape:window-maximized="1"
+     inkscape:current-layer="svg4" />
+  <path
+     d="M -1.1e-5,53.631274 C -1.1e-5,59.351949 4.622214,64 10.66666,64 H 53.333339 V 53.273744 H 10.66666 V 42.547483 h 41.955568 c 5.688889,0 11.377783,-4.648051 11.377783,-10.368726 V 32.536331 11.441344 C 64.000011,5.720669 58.666675,0 52.622228,0 H 10.66666 C 4.977766,-1.5999983e-5 -1.1e-5,5.720653 -1.1e-5,11.441327 Z M 53.677426,30.967733 H 10.322574 V 10.322569 h 43.354852 z"
+     id="path2"
+     inkscape:connector-curvature="0"
+     style="stroke-width:2.06451678"
+     sodipodi:nodetypes="ssccccsccsscssccccc" />
+</svg>
```
Comment (phallobst) [inline img/e/engadget.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 64"><path d="M0 53.6C0 59.4 4.6 64 10.7 64h42.6V53.3H10.7V42.5h42C58.2 42.5 64 38 64 32.2v.3-21A12 12 0 0 0 52.6 0h-42C5 0 0 5.7 0 11.4ZM53.7 31H10.3V10.3h43.4z"/></svg>
```

File: entries/e/engadget.com.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "Engadget": {
+    "domain": "engadget.com",
+    "url": "https://www.engadget.com/",
```
Comment (phallobst) [inline entries/e/engadget.com.json]:
```suggestion
```

File: entries/e/engadget.com.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    "tfa": [
+      "email"
+    ],
+    "documentation": "https://login.engadget.com/myaccount/security/",
```
Comment (phallobst) [inline entries/e/engadget.com.json]: The documentation is only accessible for logged-in users. Is there public documentation availlable? Alternatively, can you provide screenshots of the setup process (without any personal information)?

**Review by phallobst — CHANGES_REQUESTED** [review]

File: entries/e/engadget.com.json (hunk 3 above)
Comment (soodaayush) [reply entries/e/engadget.com.json]: I couldn't really find any public or private documentation on this feature, as it's automatically enabled when you make an account.

Comment (soodaayush) [reply entries/e/engadget.com.json]: What could I exactly do?

Comment (hkamran80) [reply entries/e/engadget.com.json]: Remove the documentation link, but attach a comment showing the setup process or its use.

Comment (soodaayush) [reply entries/e/engadget.com.json]: Changes made.

File: entries/e/engadget.com.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
+  "Engadget": {
+    "domain": "engadget.com",
+    "tfa": [
+      "email"
+    ],
+    "notes": "2FA via email is automatically set up as soon as you create your account."
+    "documentation": "",
```
Comment (hkamran80) [inline entries/e/engadget.com.json]:
```suggestion
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]
Please provide a screenshot or recording of the setup process and/or usage

File: entries/e/engadget.com.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
+{
+  "Engadget": {
+    "domain": "engadget.com",
+    "tfa": [
+      "email"
+    ],
+    "notes": "2FA via email is automatically set up as soon as you create your account."
```
Comment (hkamran80) [inline entries/e/engadget.com.json]:
```suggestion
    "notes": "2FA via email is automatically set up as soon as you create your account.",
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/e/engadget.com.json [hunk 6]
```diff
@@ -0,0 +1,12 @@
+{
+  "Engadget": {
+    "domain": "engadget.com",
+    "tfa": [
+      "email"
+    ],
+    "notes": "2FA via email is automatically set up as soon as you create your account.",
```
Comment (Carlgo11) [inline entries/e/engadget.com.json]: I don't think this qualifies as an exception or restriction.
```suggestion
```

Comment (soodaayush) [reply entries/e/engadget.com.json]: So do I just leave it as is?

Comment (Carlgo11) [reply entries/e/engadget.com.json]: Remove the note please.

**Comment by soodaayush** [comment]
engadget

This good?

**Comment by Carlgo11** [comment]
Regarding your screenshot: When choosing to log in using a one-time passcode sent to your email, do you also need to provide your password?

**Comment by soodaayush** [comment]
Nope, you don't need to provide a password. You can actually have a password-less account in Engadget, or a password protected account.

**Comment by Carlgo11** [comment]
Okay. Then Engadget isn't supporting 2FA. For it to be consider two factors, a password AND a one time code needs to be required to login. Simply replacing the password with a one time code doesn't qualify as 2FA.

**Comment by soodaayush** [comment]
To be sure, I will be enabling a password on my test account just to see if it still works.

**Comment by soodaayush** [comment]
Alright, with password enabled, it seems to stop sending codes via email, meaning that Engadget doesn't support 2FA with password enabled. Sorry for causing all of this trouble.

**Comment by phallobst** [comment]
We still could have added it as _non-supporting_...

**Pull request closed without merging**